tests/test_image_conversion.py

```diff
@@ -379,8 +379,7 @@ def test_dataset_converts_to_xarray(ij):
 
 def test_bittype_img_to_ndarray(ij):
     ArrayImgs = sj.jimport("net.imglib2.img.array.ArrayImgs")
-    dims = JArray(JLong)(3)
-    dims[:] = [10, 10, 10]
+    dims = JLong[:] @ [10, 10, 10]
```

Comment:
Better would be to use: `scyjava.jarray('j', [10, 10, 10])` because then it doesn't depend on the JPype backend.